src/command_modules/azure-cli-interactive/HISTORY.rst

```diff
@@ -3,9 +3,11 @@
 Release History
 ===============
 
-0.3.2 (unreleased)
+0.3.3 (unreleased)
 ++++++++++++++++++
+
 * --progress flag
+* Change the '?' gesture to inject into the next command
```

Comment:
If it is a change that's fine... though I think enhancing the old behavior makes sense.  For example, if you ran the command and then found out it has a lot more information that you thought it would, it would still be handy to use ? to just query and filter that output rather than having to re-run the command (and make the necessary REST calls) to filter the data.